src/ui/components/molecules/table-v2/TableV2.module.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../../globals/ads-variables.scss';
+@import '../../../../globals/fonts.scss';
+@import '../../../../globals/breakpoints';
+@import '../../../../globals/mixins';
+
+@mixin table-size($size, $gap, $max-width, $padding) {
+  &.#{$size} {
+    gap: $gap;
+    max-width: $max-width;
+    padding: $padding;
+  }
+}
+
+@mixin respond-to($breakpoint) {
```

Comment:
la funcion response-to ya esta creada en globals tienes que reutilizar esa funcion no crear otra